cli/src/main/java/org/jboss/as/cli/gui/JConsoleCLIPlugin.java

```diff
@@ -74,7 +74,7 @@
 
     // Global count of created pools
     private static final AtomicInteger executorCount = new AtomicInteger();
-    private final Logger log = Logger.getLogger(JConsoleCLIPlugin.class.getName());
+    private static final Logger log = Logger.getLogger(JConsoleCLIPlugin.class.getName());
```

Comment:
I'm not sure this is required: the `log` field is used as an instance field. We do not need to make it static.

If we do, let's rename it to `LOG` as this is our convention for constants.

Reply:
@jmesnil Thanks for your fast feedback. The pattern behind the missing static (both for LOG and the Strings below) is a final together with a direct assignment is indicating a constant. And constants shall be reused per class not per instance.
What is your thought when a immutable Object is assigned final to keep it as instance? Maybe I'm missing something.